api.planx.uk/modules/analytics/metabase/collection/types.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from "zod";
+import type { ValidatedRequestHandler } from "../../../../shared/middleware/validate.js";
+
+type ApiResponse<T> = {
+  data?: T;
+  error?: string;
+};
+
+export const checkCollectionsSchema = z.object({
+  query: z.object({
+    name: z.string().min(1, "Name parameter is required"),
+  }),
+});
+
+export type CheckCollectionsRequest = ValidatedRequestHandler<
+  typeof checkCollectionsSchema,
+  ApiResponse<CheckCollectionResponse>
+>;
+
+export interface CheckCollectionResponse {
+  collectionId: number | false;
+}
+
+export interface NewCollectionParams {
```

Comment:
Metbase docs suggest that the params `description`, `namespace` or `authority_level` are not required, and we likely won't be configuring them.

If this is the case we should remove them from the type and not worry about them.